package.json

```diff
@@ -49,6 +49,9 @@
     "@fortawesome/free-solid-svg-icons": "^5.3.1",
     "@fortawesome/react-fontawesome": "^0.1.2",
     "axios": "^0.19.0",
+    "@material-ui/core": "^4.0.1",
+    "@material-ui/icons": "^4.0.1",
+    "@material-ui/lab": "^4.0.0-alpha.14",
```

Comment:
I'm wondering if we add these as `peerDependencies`, instead of `dependencies`, since Avalon might already pick up `@material-ui/...` from the Timeliner?

Reply:
I think the timeliner is pretty self-contained right now in its own pack so I don't think it is pulled in through package.json.

Reply:
Ok, @cjcolvar I was wondering how things are compiling within Avalon.  If by webpack, then I'm guessing webpack will recognize transitive dependencies (`material-ui`) from both Timeliner and SME, and only include one version?

Reply:
Unless they both specify different versions of `material-ui`, then it will include both I think.

Reply:
If anything I think the SME's material ui might leak into the timeliner page but not the other way around since the timeliner is statically built into a single pack that is only included on the timeliner.

Reply:
Ah, got it.